.github/workflows/packaging.yaml

```diff
@@ -0,0 +1,216 @@
+name: Package linux binaries
+
+env:
+  MY_LIST: "0.10.7,0.13.6,0.14.1,0.16.1,0.17.3,0.18.3,0.19.4,0.21.1,0.24.0,0.26.5,0.28.0,0.29.1,0.31.2,0.32.2"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  package-binaries:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        arch: [amd64]
+
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+
+    steps:
+
+      - name: Configure AWS credentials
+        run: |
+          cd $HOME; mkdir ~/.aws; touch ~/.aws/credentials; touch ~/.aws/config
+          echo "[default]
+          aws_access_key_id = ${{ secrets.AWS_ACCESS_KEY_ID_CF }}
+          aws_secret_access_key = ${{ secrets.AWS_SECRET_ACCESS_KEY_CF }}" > ~/.aws/credentials
+          echo "[default]
+          region=auto
+          output=json"  > ~/.aws/config
+          
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+          submodules: recursive
+
+      - name: Install prerequisites
+        run: |
+          sudo apt-get install -y gcc dpkg-dev gpg
+
+      - name: Create folder structures for Linux
+        run: |
+          for version in $(echo $MY_LIST | tr "," "\n"); do
+          mkdir -p ~/linux/axelard_$version-1_amd64/usr/bin/
+          mkdir -p ~/linux/axelard_$version-1_amd64/DEBIAN/
+          mkdir -p ~/linux/axelard_$version-1_arm64/usr/bin/
+          mkdir -p ~/linux/axelard_$version-1_arm64/DEBIAN/
+          mkdir -p ~/linux/axelard_$version-1_arm/usr/bin/
+          mkdir -p ~/linux/axelard_$version-1_arm/DEBIAN/   
+          done 
+          
+      - name: Get binaries
+        run: |
+          for version in $(echo $MY_LIST | tr "," "\n"); do
+          wget -O ~/linux/axelard_$version-1_amd64/usr/bin/axelard --no-check-certificate https://github.com/axelarnetwork/axelar-core/releases/download/v$version/axelard-linux-amd64-v$version
+          chmod 755 ~/linux/axelard_$version-1_amd64/usr/bin/axelard
+          wget -O ~/linux/axelard_$version-1_arm64/usr/bin/axelard --no-check-certificate https://github.com/axelarnetwork/axelar-core/releases/download/v$version/axelard-linux-arm64-v$version
+          chmod 755 ~/linux/axelard_$version-1_arm64/usr/bin/axelard
```

Comment:
why do we have version-1 ?

Reply:
it is not mandatory but it is a convention for the package name : 
<package-name>_<version>-<release-number>_<architecture>
The release number is used to track different releases of the same software version; usually it is set to 1, in case of error in the packaging, the number would be increased